src/renderer/screens/FirmwareUpdate.js

```diff
@@ -66,10 +67,15 @@ const FirmwareUpdate = (props) => {
   const [confirmationOpen, setConfirmationOpen] = useState(false);
   const [resetOnFlash, setResetOnFlash] = useState(false);
   const [activeStep, setActiveStep] = useState(-1);
+  const [flashSteps, setFlashSteps] = useState([]);
   const { t } = useTranslation();
   const focusDeviceDescriptor =
     props.focusDeviceDescriptor || focus.focusDeviceDescriptor;
 
+  useEffectOnce(() => {
```

Comment:
This wants some comments inline about why we only want to do this on first render and why having it run on every rerender would be bad (presumably because props.focusDeviceDescriptor gets reset?)

Reply:
ACK, adding. And yep, because both `props.focusDescriptor` and `focus.focusDescriptor` end up being null after the reconnect step, at least temporarily, and we don't want even a temporary flicker.